src/XamlStyler.dotUltimate/buildPlugin.ps1

```diff
@@ -0,0 +1,10 @@
+Param(
```

Comment:
This file is used for building a R# plugin. Given you already have a VS plugin, I did not wire this up in the Azure DevOps pipeline. If R# is not needed, this file can be dropped.

Reply:
OK, will drop it. Same for runVS.ps1 and publish plugin.ps1, those are all for the R# plugin.

Reply:
Dropped with next commit.